src/commands.cpp

```diff
@@ -2688,6 +2688,35 @@ void CommandDelUA::procresult()
 
 #endif
 
+
+
+CommandGetUserEmail::CommandGetUserEmail(MegaClient *client, const char *uid)
+{
+    cmd("uge");
```

Comment:
`this->client = client;` could be added here, like in other commands.
It's not strictly needed because the SDK already sets it externally before calling `procresult()` but it seems a bit more logical that the command saves the values it needs rather than relying on the external access to its attributes.